schema/defs/gkrule/CanonicalVariationRule.rst

```diff
@@ -0,0 +1,34 @@
+**Computational Definition**
+
+The CanonicalVariationRule class constructs objects which describe any variation that are congruent with the indicated *variation* value. The rules by which congruency is determined are managed at the implementation level, but typically involve operations such as variation projection (genomic <-> transcript), translation (transcript <-> protein) and liftover (e.g. GRCh37 genomic <-> GRCh38 genomic).
```

Comment:
I'm thinking we may want to use more appropriate and less technical terms to represent the subtypes of Simple and Complex Variation Rules.
In the VRS spec we don't force our subtypes to contain the superclasses name by default (at least not as a hard rule). 
So I would propose that we call this particular subclass `CanonicalVariation` which is inherently a VariationRule. I think the usefulness of these more thoughtful terms will become apparent as we look at the other subtypes.

Reply:
forget this. we can address later if necessary. I'm getting a bit confused by our naming approach. Should the class name include the type of VRS object it measures (Variation, Location, Sequence, Gene, Feature, etc...) or should it contain the core elements that the rule uses to do the measuring to identify the VRS objects that it will output. Or both?  Will we have LocationRules and VariationRules, etc..? If so, we should try to define a rational approach to how we name these "categorical" classes.